src/handlers/routeHandler.js

```diff
@@ -0,0 +1,17 @@
+const getIdFreeUrl = url => {
```

Comment:
not sure if it's good approach. You can have a lot of path parameters, which divided by `/` and they can appear in different places `/users/<userId>/products/<productId>`